internal/controllers/app_controller.go

```diff
@@ -583,13 +584,15 @@ func (r *AppReconciler) watchFunc(ctx context.Context, cleanup cleanupFunc, app
 				recorder.AnnotatedEventf(app, appDeploymentEvent.Annotations, v1.EventTypeNormal, ketchv1.AppReconcileUpdate, appDeploymentEvent.Description)
 			}
 		case <-healthcheckTimeout:
+			podName, _ := checkPodStatus(r.Group, r.Client, app.Name, app.Spec.Deployments[len(app.Spec.Deployments)-1].Version)
```

Comment:
more for learning on my side, but why is there no need to check if error != nil for the following cases?